azurerm/internal/services/mssql/parse/mssql.go

```diff
@@ -29,6 +29,18 @@ type MsSqlDatabaseExtendedAuditingPolicyId struct {
 	ResourceGroup string
 }
 
+func NewMsSqlDatabaseID(resourceGroup, msSqlServer, name string) MsSqlDatabaseId {
```

Comment:
Should this be a more descriptive name?

```suggestion
func SynapseSqlPoolDatabaseID(resourceGroup, msSqlServer, name string) MsSqlDatabaseId {
```